client/extensions/woocommerce/app/settings/email/mailchimp/setup-steps/newsletter-settings.js

```diff
@@ -21,8 +21,12 @@ const NewsletterSettings = ( { storeData = {}, onChange, siteId, isRequesting, t
 	return (
 		<FormFieldset className="setup-steps__store-info-field">
 			<QueryMailChimpLists siteId={ siteId } />
-			<div>{ translate( 'Pick a list, you will not able to change it for now so pick carefully.' ) }</div>
-			<div>{ translate( 'Create your list at mailchimp.com if you have not done it aready.' ) }</div>
+			<div className="setup-steps__login-title">{ translate( 'Now choose a list to sync!' ) }</div>
+			<p>{ translate(
+					'Choose your list carefully as you wont be able to change it later. ' +
```

Comment:
I'm not 💯 positive but thinking having these strings concatenated here might not be the proper way to do the translation call. Was this just to keep the link a certain length?

Reply:
Kelly did this bit - maybe she knows more. I think that it is OK. 
@kellychoffman

Reply:
Yeah, its to keep the line length shorter to keep the calypso bots happy. I believe its done in other places like this, but happy to change it if there's a better way to do it.

Reply:
@timmyc @budzanowski @kellychoffman We asked in the Calypso channel awhile back and this is totally fine for translation calls.

Reply:
:TIL: thanks!